gcloud-java-compute/src/test/java/com/google/gcloud/compute/SerializationTest.java

```diff
@@ -105,12 +105,8 @@
       .maintenanceWindows(WINDOWS)
       .region(REGION_ID)
       .build();
-  private static final Long DELETED = 1453293540000L;
-  private static final Long DEPRECATED = 1453293420000L;
-  private static final Long OBSOLETE = 1453293480000L;
   private static final DeprecationStatus<MachineTypeId> DEPRECATION_STATUS =
-      new DeprecationStatus<>(DELETED, DEPRECATED, OBSOLETE, MACHINE_TYPE_ID,
-          DeprecationStatus.Status.DELETED);
+      DeprecationStatus.of(DeprecationStatus.Status.DELETED, MACHINE_TYPE_ID);
```

Comment:
New classes in this PR that implement Serializable (e.g. `ImageInfo`) can be added to this file.